src/useCases/storeVerificationCode.ts

```diff
@@ -6,7 +6,8 @@ export default async (connection: Database, emailAddress: string, verificationCo
   const storeVerificationQuery = `
```

Comment:
I just noticed that we don't have a test for this use case. It would be great if write some tests for it in this PR.